maptivist/src/main/webapp/script.js

```diff
@@ -181,79 +110,74 @@ function inBoundary(marker){
   var bounds = map.getBounds().toJSON();
   return (marker.latitude <= bounds.north && 
          marker.latitude >= bounds.south &&
-         marker.longitude <= bounds.west &&
-         marker.longitude >= bounds.east);
+         marker.longitude >= bounds.west &&
+         marker.longitude <= bounds.east);
 }
 
 /** Adds a new Marker based on id array
  */
```

Comment:
Can you update this comment?

The input value is a Marker in json format, right?

Reply:
Yes that's correct let me make that change